ow_faults/msg/ArmFaults.msg

```diff
@@ -0,0 +1,12 @@
+# Arm Fault Status 
+Header header
+uint32 value
+# None = 0
+# Hardware = 1
+# Trajectory Generation = 2
+# Collision = 3
+# Estop = 4
+# Position Limit = 5
+# Torque Limit = 6
+# Velocity Lmit = 7
```

Comment:
typo Lmit

Reply:
And a more substantial question: are any other than Hardware implemented?  I get that one when I fault an encoder.

Reply:
Nope, only hardware right now. JPL hasn't really implemented others beyond hardware and we also decided to focus first on hardware anyways.